ui/public/locales/en.json

```diff
@@ -675,9 +675,9 @@
 "label.deploymentplanner": "Deployment planner",
 "label.desc.db.stats": "Database Statistics",
 "label.desc.importexportinstancewizard": "Import and export Instances to/from an existing VMware or KVM cluster.",
-"label.desc.import.ext.kvm.wizard": "Import libvirt domain from KVM Host",
-"label.desc.import.local.kvm.wizard": "Import QCOW image from Local Storage",
-"label.desc.import.shared.kvm.wizard": "Import QCOW image from Shared Storage",
+"label.desc.import.ext.kvm.wizard": "Import Instance from remote KVM host",
```

Comment:
is it really "remote"? this is a configured host in cloudstack is it?

Reply:
No, it's an external host (not part ACS, needs to run libvirt i.e. also NOT proxmox host) -  so... hm it can be either "remote" or  "external" KVM host - opinions?

Reply:
ok, no is good never mind